master/nightly_noinline.py

```diff
@@ -0,0 +1,73 @@
+###############################################################################
+# Define everything needed to build nightly Julia for noinline test
+###############################################################################
+
+noinline_nightly_scheduler = Nightly(name="Julia Noinline Build", builderNames=["nightly_noinline-x64"], hour=[3], branch="master", onlyIfChanged=True)
+c['schedulers'].append(noinline_nightly_scheduler)
+
+arch = "x86"
```

Comment:
I make it run only on 64bit to save some time but deleted the wrong one here...... fixed....